### ISSUE TITLE
Question Spawn

### ISSUE THREAD

**Issue by corentinbost** [lenis0012/LoginSecurity#319]
Hello, is it possible at the Spawn teleport location to teleport to Minecraft coordinates?  (example: 231, 79, 195)

**Comment by vaminecHUN** [comment]
dowload essentials spawm 😁 and you will have/spawn command too :D

**Comment by lenis0012** [comment]
Yeah you have to use a plugin that allows you to change the world spawn location such as Essentials or MyWorlds
